graphqlapi/src/main/java/io/stargate/graphql/web/GraphqlApiServerFactory.java

```diff
@@ -61,7 +60,6 @@ public Server build(Environment environment) {
     final ContextRoutingHandler routingHandler = new ContextRoutingHandler(handlers);
     final Handler gzipHandler = buildGzipHandler(routingHandler);
     server.setHandler(addStatsHandler(addRequestLog(server, gzipHandler, environment.getName())));
-
     return server;
```

Comment:
Nit: unnecessary changes (probably a side-effect of something that was reverted).